packages/react/src/components/DashboardEditor/DashboardEditor.jsx

```diff
@@ -26,7 +27,7 @@ const propTypes = {
   title: PropTypes.node,
   /** initial dashboard data to edit */
   initialValue: PropTypes.shape({
-    cards: PropTypes.array,
+    cards: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Ideally, this one needs to be more fleshed out, but the CardPropTypes need some work to be of more help here. I can't just use `PropTypes.oneOfType([CardPropTypes, BarChartCardPropTypes])` for example, because the BarChartCardPropTypes don't actually include _all_ the card PropTypes they're just smashed together with a spread operation.